diff --git a/sm/csm/laser_data.c b/sm/csm/laser_data.c
index 7f47fb7..52e2a1b 100644
--- a/sm/csm/laser_data.c
+++ b/sm/csm/laser_data.c
@@ -193,13 +193,13 @@ int ld_valid_fields(LDP ld)  {
 			ld->min_theta, ld->max_theta);
 		return 0;
 	}
-	double min_fov = deg2rad(20.0); 
+/*	double min_fov = deg2rad(20.0);
 	double max_fov = 2.01 * M_PI;
 	double fov = ld->max_theta - ld->min_theta;
 	if( fov < min_fov || fov > max_fov) {
 		sm_error("Strange FOV: %f rad = %f deg \n", fov, rad2deg(fov));
 		return 0;
-	}
+	}*/
 	if(fabs(ld->min_theta - ld->theta[0]) > 1e-8) {
 		sm_error("Min_theta (%f) should be theta[0] (%f)\n",
 			ld->min_theta, ld->theta[0]);
